src/ontology/mondo-edit.obo

```diff
@@ -544563,6 +544563,38 @@ intersection_of: MONDO:0005336 ! myopathy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/29456 ! TOR1AIP1
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8593" xsd:anyURI
 
+[Term]
+id: MONDO:0100587
+name: A4GALT-CDG
+def: "Any congenital disorder of glycosylation in which the cause of the disease is a mutation in A4GALT." [https://orcid.org/0009-0005-6507-3838, https://www.clinicalgenome.org/affiliation/40135/, PMID:30454869]
+synonym: "A4GALT-congenital disorder of glycosylation" EXACT [PMID:30454869]
+synonym: "A4GALT-related congenital disorder of glycosylation" EXACT [https://orcid.org/0000-0001-5208-3432]
+is_a: MONDO:0015286 {source="https://www.clinicalgenome.org/affiliation/40135/"} ! congenital disorder of glycosylation
+intersection_of: MONDO:0015286 ! congenital disorder of glycosylation
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/18149
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8523" xsd:anyURI
+
+[Term]
+id: MONDO:0100588
+name: FAM20B-CDG
+def: "Any congenital disorder of glycosylation in which the cause of the disease is a mutation in FAM20B." [PMID:30847897, PMID:37298583, PMID:38653092]
+synonym: "FAM20B-congenital disorder of glycosylation" EXACT [PMID:30847897, PMID:37298583, PMID:38653092]
+is_a: MONDO:0015286 {source="https://www.clinicalgenome.org/affiliation/40135/"} ! congenital disorder of glycosylation
+intersection_of: MONDO:0015286 ! congenital disorder of glycosylation
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23017
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8525" xsd:anyURI
+
+[Term]
+id: MONDO:0100589
+name: ALG10-CDG
+def: "Any congenital disorder of glycosylation in which the cause of the disease is a mutation in ALG10." [PMID:33798445]
+synonym: "ALG10-congenital disorder of glycosylation" EXACT [https://orcid.org/0009-0005-6507-3838, https://www.clinicalgenome.org/affiliation/40135/, PMID:33798445]
+is_a: MONDO:0015286 {source="https://www.clinicalgenome.org/affiliation/40135/"} ! congenital disorder of glycosylation
+intersection_of: MONDO:0015286 ! congenital disorder of glycosylation
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23162 ! ALG10
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/23162 ! ALG10
```

Comment:
source is missing

Reply:
I removed the subclass axiom because it was redundant